jsons/Nations.json

```
@@ -1064,6 +1064,41 @@
         "spyNames": [
             "Johann","Marlene","Wilhelm","Eva","Heinz","Horst","Carl","Viper","Albrecht","Anton"
         ]
+    },
+	{
+        "name": "Georgia",
+        "leaderName": "Tamar",
+        "adjective": ["Georgian"],
+        "startBias": ["Avoid [Coast]"],
+        "startIntroPart1": "The mountains echo with the strength of our ancestors.",
+        "startIntroPart2": "Georgia rises, proud and unbroken.",
+        "declaringWar": "Your actions dishonor the legacy of peace. Now, feel the might of Georgia!",
+        "attacked": "Your treachery knows no bounds! Georgia will endure, and you will fall!",
+        "defeated": "The proud heart of Georgia is silent... but her spirit will rise again.",
+        "introduction": "Gamarjoba. In the name of the ancient land of Kartli, welcome.",
+        "neutralHello": "The mountains stand still. What do you seek?",
+        "hateHello": "Enough talk! What is it this time?",
+        "tradeRequest": "This trade honors us both. Shall we proceed?",
+        "outerColor": [147,112,219],
+        "innerColor": [246,205,137],
+        "uniqueName": "Ecclesiastical Architecture",
+        "uniques": [
+            "[+2 Production] from every [Gurdwara]",
+            "[+2 Production] from every [Cathedral]",
+            "[+2 Production] from every [Mandir]",
+            "[+2 Production] from every [Mosque]",
+            "[+2 Production] from every [Pagoda]",
+            "[+2 Production] from every [Synagogue]",
+            "[+2 Production] from every [Vihara]"
+            // TODO: new uniq Buildings unlocked by religious beliefs (except Houses of Worship) are built using Production instead (cost equal to 50% of the required Faith)
+        ],
+        "cities": [
+            "Tbilisi","Batumi","Kutaisi","Sokhumi","Zugdidi","Rust'avi","P'ot'i","Gori","Ts'khinvali","Samtredia","Khashuri","Senak'i","Zestap'oni",           "Telavi","Akhaltsikhe","Ozurgeti","Kaspi","Chiatura","Tskaltubo","Sagarejo","Gardabani","Borjomi","Tkibuli","Khoni","Bolnisi","Akhalkalaki","Gurjaani",
+            "Mtskheta","Kvareli","Akhmeta","Kareli","Lanchkhuti","Tsalenjikha","Dusheti","Sachkhere","Dedoplistsqaro"
```

Sorry for the delay here
```suggestion
            "Tbilisi","Kutaisi","Batumi","Sokhumi","Zugdidi","Rust'avi","P'ot'i","Gori","Ts'khinvali","Samtredia",
            "Khashuri","Senak'i","Zestap'oni","Telavi","Ozurgeti","Marneuli","Kobuleti","Tqvarch'eli","Akhaltsikhe","Tsqaltubo",
            "Kaspi"
```
Apparently their name list is pretty small... huh

---

Btw, If you want to know where I find the lists, the "list" of names is in a big table somewhere in CivUnits.xml, and looks something like
```xml
		<Row> 
			<CivilizationType>CIVILIZATION_GEORGIA</CivilizationType> 
			<CityName>TXT_KEY_CITY_NAME_GEORGIA_1</CityName> 
		</Row>
		<Row> 
			<CivilizationType>CIVILIZATION_GEORGIA</CivilizationType> 
			<CityName>TXT_KEY_CITY_NAME_GEORGIA_2</CityName> 
		</Row>
```
You actually need to check CivUnits_Mongol.xml for the actual texts
```xml
	<Row Tag="TXT_KEY_CITY_NAME_GEORGIA_0">			
		<Text>Tbilisi</Text>
	</Row>
	<Row Tag="TXT_KEY_CITY_NAME_GEORGIA_1">			
		<Text>Kutaisi</Text>
	</Row>
	<Row Tag="TXT_KEY_CITY_NAME_GEORGIA_2">			
		<Text>Batumi</Text>
	</Row>
```
A similar story is true for spy names, though looks like you have them all I think

---

I mean, if you want, you could just adding the new names after the "official" ones to reach 30 or so names